src/mixins/itext_behavior.mixin.js

```diff
@@ -512,6 +506,7 @@
         e.dataTransfer.effectAllowed = 'copyMove';
         this.setDragImage(e, data);
       }
+      this.abortCursorAnimation();
```

Comment:
for future follow up
this line might be causing selection to be cleared on drag start